### ISSUE TITLE
Options reset to defaults

### ISSUE THREAD

**Issue by notuntoward** [gildas-lormeau/SingleFile#1412]
This morning, I noticed that all my SingleFile options had been reset to the defaults -- things like the saved filename, infobar contents, html type, etc.  I know that I used SingleFile a couple of days ago, and then, those options were still as I'd set them many months ago.

**To Reproduce**
I'm not sure what caused this, but I know that the Brave browser updated itself recently

**Environment**
 - OS: Windows 11 and MacOS Sonoma
 - Browser: Brave 
 - Version: v1.63.169

**Comment by gildas-lormeau** [comment]
Thank you, I have 2 questions. Is SingleFile the only extension to have lost data? Have you activated the synchronization option in the options page?

**Comment by notuntoward** [comment]
Yes, that's the only one that lost data.  I'm also syncrhonizing SingleFile settings across Brave browsers, one on MacOS and one on Windows.
